jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/log/RuntimeLogPresenter.java

```diff
@@ -53,60 +65,117 @@ public IsWidget getWidget() {
+                                           logType))
+                    .filter(Objects::nonNull)
+                    .collect(toList());
+
+            view.setLogs(logsLine);
+        }).getProcessInstanceLogs(currentServerTemplateId,
+                                  currentDeploymentId,
+                                  currentProcessInstanceId);
+    }
+
+    protected String getLogline(RuntimeLogSummary logSummary,
+                                LogType logType) {
 
         if (LogType.TECHNICAL.equals(logType)) {
-            processRuntimeDataService.call(new RemoteCallback<List<RuntimeLogSummary>>() {
-                @Override
-                public void callback(List<RuntimeLogSummary> logs) {
-                    final List<String> logsLine = new ArrayList<String>();
-
-                    if (logOrder == LogOrder.DESC) {
-                        Collections.reverse(logs);
-                    }
-
-                    for (RuntimeLogSummary rls : logs) {
-                        logsLine.add(rls.getTime() + ": " + rls.getLogLine() + " - " + rls.getType());
-                    }
-
-                    view.setLogs(logsLine);
-                }
-            }).getRuntimeLogs(currentServerTemplateId,
-                              currentDeploymentId,
-                              currentProcessInstanceId);
+            String agent = constants.System();
+            if ((NODE_HUMAN_TASK.equals(logSummary.getNodeType()) && logSummary.isCompleted()) ||
+                    (NODE_START.equals(logSummary.getNodeType()) && !logSummary.isCompleted())) {
+                agent = constants.Human();
+            }
+            return format(TECH_LOG_FORMAT,
+                          DateUtils.getDateTimeStr(logSummary.getDate()),
+                          logSummary.getNodeName(),
+                          logSummary.getNodeType(),
+                          (logSummary.isCompleted() ? " " + constants.Completed() : ""),
+                          agent
+            );
         } else {
-            processRuntimeDataService.call(new RemoteCallback<List<RuntimeLogSummary>>() {
-                @Override
-                public void callback(List<RuntimeLogSummary> logs) {
-                    final List<String> logsLine = new ArrayList<String>();
-                    if (logOrder == LogOrder.DESC) {
-                        Collections.reverse(logs);
-                    }
-
-                    for (RuntimeLogSummary rls : logs) {
-                        logsLine.add(rls.getTime() + ": " + rls.getLogLine());
-                    }
-
-                    view.setLogs(logsLine);
-                }
-            }).getBusinessLogs(currentServerTemplateId,
-                               currentDeploymentId,
-                               currentProcessName,
-                               currentProcessInstanceId);
+            String prettyTime = moment(DateTimeFormat.getFormat("yyyy-MM-dd hh:mm:ss a")
```

Comment:
See my comment on the other PR, just wondering if you could create the moment object without parsing the date, either by using the date object or date..getTime()